collections/guides/own_and_use_us_bank_accounts/index.md

```diff
@@ -0,0 +1,134 @@
+
+```js 
+  await BasisTheory.init(ELEMENTS_KEY, {
+    elements: true,
+  });
+
+  let routingNumber = BasisTheory.elements.create('text', {
+    ...options,
+    targetId: 'routingNumber',
+    placeholder: 'Routing Number',
+    'aria-label': 'Routing Number',
+  });
+  let accountNumber = BasisTheory.elements.create('text', {
+    ...options,
+    targetId: 'accountNumber',
+    placeholder: 'Account Number',
+    'aria-label': 'Account Number',
+  });
+
+  await routingNumber.mount('#routing_number');
+  await accountNumber.mount('#account_number');
+```
+
+Finally, you'll see two new Elements appear on your page. Next, we will tokenize the data entered into them!
+
+## Create an Atomic Bank from your new Elements
+This is where our Elements shine, you'll interact with our SDK as if you're passing raw text. 
+
+To create an [Atomic Bank](https://docs.basistheory.com/api-reference/#atomic-banks), you'll just need to pass the Element references from Step 1 into the `atomicBank.create` method.
+
+```js
+const token = await BasisTheory.elements.atomicBank.create({
+      bank: {
+        routingNumber,
+        accountNumber,
+      },
+    });
+```
+
+Once completed, you'll have a new Atomic Bank assigned to `token`.
+
+## Set up a Spreedly Reactor
+
+Configure one of our pre-built Reactor Formulas, enabling you to quickly exchange raw card data for a processor token (e.g. Spreedly - Bank).
+
+To set up a Reactor, head over to our Portal and set up a new Stripe Reactor. If you're looking for step-by-step guide on how to set up a Reactor, head over to our [guide](/guides/setup-your-first-reactor).
+
+## Using your new Reactor 
+<span class="base-alert warning">
+  <span>
+    To use your new Spreedly Reactor, you'll need a <code>server-to-server</code> application with the following permissions <code>bank:read</code>, <code>bank:create</code>, and <code>reactor:read</code>.
+  </span>
+</span>
+
+Once you’ve created your Spreedly Reactor, use the <code>reactor_id</code> and your Atomic Bank's ID to exchange for a new Spreedly Payment Method token, enabling you to charge your customer.
+
+```js
+const reactionToken = await bt.atomicBanks.react(token.id, {
+    reactorId: REACTOR_ID
+});
+
+return reactionToken.data.id;
+```
+
+With your Spreedly Payment Method in hand, you can store this within your own platform and avoid becoming dependent on Basis Theory for your future transactions. As soon as you need a new Payment Method, just call the `/react` endpoint for that bank, and you'll have a newly attached token.
+
+
+## Wrap up
+
+Now that you understand the basics of using our Elements and Atomic Banks to collect and charge a customer, you have the ability to easily use your encrypted Atomic Bank in any way you'd like:
+
+- Use our Serverless Reactors to create an ACH file (if you're looking to do this, [reach out!](https://basistheory.com/contact))
+- Quickly begin securing and using raw bank data yourself with our API

```

Comment:
want to embed this loom video once rad has it on youtube. https://www.loom.com/share/67143f3fd18f41a3a70a8725f07099d8